crates/camera/src/camera.rs

```diff
@@ -23,13 +23,17 @@ const CAMERA_HORIZONTAL_SPEED: InverseSecond = Quantity::new_unchecked(2.0);
 /// Minimum camera distance from terrain achievable with zooming along.
 const MIN_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(20.);
 /// Maximum camera distance from terrain achievable with zooming alone.
-const MAX_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(100.);
+const MAX_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(300.);
+/// Maximum camera distance from terrain achievable with zooming alone.
+const DEFAULT_CAMERA_DISTANCE: Metre = Quantity::new_unchecked(
+    (MIN_CAMERA_DISTANCE.get() + (MAX_CAMERA_DISTANCE.get() - MIN_CAMERA_DISTANCE.get())) * 0.7,
```

Comment:
I would make the default smaller (0.4?). Since currently the high zoom out mode is far from perfect (no LOD, missing switch to cartographic "mode" etc.).